File: app/javascript/channels/meeting_channel.js
import consumer from "./consumer"

document.addEventListener('turbolinks:load', () => {
  const meetingElement = document.getElementById('meeting-id')
  if (meetingElement) {
    const meetingId = meetingElement.getAttribute('data-meeting-id')
    consumer.subscriptions.create(
      { channel: "MeetingChannel", meeting_id: meetingId },
      {
        received(data) {
          console.log("Received data:", data)
          // ここでDOMを更新するなど
        }
      }
    )
  }
})

File: app/channels/meeting_channel.rb
# app/channels/meeting_channel.rb
class MeetingChannel < ApplicationCable::Channel
  def subscribed
    # 接続時に受け取ったパラメータ meeting_id を購読
    stream_from "meeting_#{params[:meeting_id]}"
  end

  def unsubscribed
    # 何も書かなくてもOK
  end
end
